services/blockchain-indexer/tests/unit/shared/dataservice/business/transactionEstimateFees.test.js

```diff
@@ -46,59 +54,88 @@ describe('Test transaction fees estimates', () => {
 		};
 	});
 
-	const {
-		mockTransaction,
-		calcDynamicFeeEstimates,
-	} = require('../../../../../shared/dataService/business/transactionsEstimateFees');
-
 	describe('Test transaction fees estimate', () => {
+		beforeEach(() => {
+			// jest.resetAllMocks();
+			// jest.resetModules();
+		});
+
```

Comment:
don't we need to reset mocks before tests? incase it is better not to reset, maybe we can remove these comments?